index.ts

```diff
@@ -185,6 +185,19 @@ import {
 } from "./schemas.js";
 import { randomUUID } from "crypto";
 
+const pino = require('pino')
```

Comment:
This fails on my machine with:
```
const pino = require('pino');
             ^

ReferenceError: require is not defined in ES module scope, you can use import instead
This file is being treated as an ES module because it has a '.js' file extension and '<redacted>/gitlab-mcp/package.json' contains "type": "module". To treat it as a CommonJS script, rename it to use the '.cjs' file extension.
```